.ci/no-exception-test.sh

```diff
@@ -1,31 +1,7 @@
 #!/bin/bash
 set -e
 
-removeFolderWithProtectedFiles() {
-  find $1 -delete
-}
-
-function getCheckstylePomVersion {
-  echo "$(mvn -e --no-transfer-progress -q -Dexec.executable='echo' \
-                      -Dexec.args='${project.version}' \
-                      --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)"
-}
-
-function checkout_from {
-  CLONE_URL=$1
-  PROJECT=$(echo "$CLONE_URL" | sed -nE 's/.*\/(.*).git/\1/p')
-  mkdir -p .ci-temp
-  cd .ci-temp
-  if [ -d "$PROJECT" ]; then
-    echo "Target project $PROJECT is already cloned, latest changes will be fetched"
-    cd $PROJECT
-    git fetch
-    cd ../
-  else
-    for i in 1 2 3 4 5; do git clone $CLONE_URL && break || sleep 15; done
-  fi
-  cd ../
-}
+source ./util.sh
```

Comment:
This should be `source ./.ci/util.sh`

Reply:
and for travis.sh, what's the correct way?

Reply:
Same, this will work for travis also. You can run locally to check changes. `./.ci/travis/travis.sh test` .

Reply:
done

Reply:
Resolved.